2.10 Refresh 03/27 - 03/28

dockerymick opened the pull request.

xenolinux committed:
ACM#9499: Document the hosting cluster capacity metrics

xenolinux committed:
ACM#9499: Override resource utilization values and disable monitoring

swopebe committed:
https://issues.redhat.com/browse/ACM-9744--what's new and Epic review

swopebe committed:
https://issues.redhat.com/browse/ACM-9744--what's new and Epic review

dockerymick committed:
Update whats_new.adoc

suggested updates after scanning w/Brandi

dockerymick committed:
Update whats_new.adoc

xenolinux committed:
Merge pull request #6233 from stolostron/metrics-hcp-210

ACM#9499: Document the hosting cluster capacity metrics

swopebe committed:
Merge pull request #6266 from stolostron/dockerymick-patch-1

https://issues.redhat.com/browse/ACM-9744 Update whats_new.adoc

Commit:
added sample yaml for gitops

swopebe committed:
https://issues.redhat.com/browse/ACM-10550--known issue rewritten, edited, added

jc-berger committed:
Update gitops/gitops_push_pull.adoc

Co-authored-by: Mikela Jackson <mdockery@redhat.com>

jc-berger committed:
Merge pull request #6269 from stolostron/jcberger-10457-example-gitops

added sample yaml for gitops

swopebe committed:
Merge pull request #6270 from stolostron/swope-known-issue4

https://issues.redhat.com/browse/ACM-10550--known issue rewritten, ed…

mprahl committed:
Fix the Governance RBAC documentation

Fixes:
https://issues.redhat.com/browse/ACM-10602

Signed-off-by: mprahl <mprahl@users.noreply.github.com>

omertuc committed:
ACM-10596: Address custom API domain import (#6078)

Clusters using a custom API domain with custom certificates need special
treatment when imported

dockerymick committed:
Updating links for Gov after 2.10 GA

swopebe committed:
Merge pull request #6274 from stolostron/mj-fix-link-postga

Updating links for Gov after 2.10 GA

Commit:
removed cluster backup release note for 2.10

jc-berger committed:
Merge pull request #6275 from stolostron/jcberger-10601-remove-cluster-backup-release-note-2.10

removed cluster backup release note for 2.10

dockerymick committed:
https://issues.redhat.com/browse/ACM-10052 (#6268)

* https://issues.redhat.com/browse/ACM-10052

* Update api.adoc

* Update compliance_history.adoc

* Update compliance_history.adoc

Updating new topic

* Update compliance_history.adoc

* Update compliancehistory.json.adoc

some updates

* Update compliancehistory.json.adoc

* Apply suggestions from code review

Co-authored-by: jc-berger <70717303+jc-berger@users.noreply.github.com>
Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update compliancehistory.json.adoc

* Update apis/compliancehistory.json.adoc

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

* Update compliancehistory.json.adoc

active voice, eliminate to be

* Update compliance_history.adoc

adding the Configuring PostgreSQL

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update apis/compliancehistory.json.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update compliance_history.adoc

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

* Update apis/compliancehistory.json.adoc

* Update apis/compliancehistory.json.adoc

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update compliance_history.adoc

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update apis/compliancehistory.json.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

* Update governance/compliance_history.adoc

Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>

---------

Co-authored-by: jc-berger <70717303+jc-berger@users.noreply.github.com>
Co-authored-by: Matt Prahl <mprahl@users.noreply.github.com>
Co-authored-by: swope <bswope@redhat.com>

dockerymick committed:
Updates for 2.10 refresh

swopebe committed:
Merge pull request #6278 from stolostron/mj-fix-books-2.10

Updates for 2.10 refresh

dockerymick committed:
Making updates (#6280)

dockerymick committed:
Adding What's new entry for feature (#6281)

oafischer committed:
https://issues.redhat.com/browse/ACM-10170 Remove YAML switch

oafischer committed:
https://issues.redhat.com/browse/ACM-10170

oafischer committed:
https://issues.redhat.com/browse/ACM-10507 Fix label typo

oafischer committed:
Fix typo

Commit:
volysync version change

oafischer committed:
https://issues.redhat.com/browse/ACM-10170

oafischer committed:
https://issues.redhat.com/browse/ACM-10170

Commit:
removed hidden comments and other fixes

jc-berger committed:
Merge pull request #6285 from stolostron/jcberger-10400-volysync-version

volysync version change

swopebe committed:
Update release_notes_intro.adoc wrong version in removal

swopebe committed:
https://issues.redhat.com/browse/ACM-10400 Update release_notes_intro.adoc

swopebe committed:
Update release_notes_intro.adoc

swopebe committed:
https://issues.redhat.com/browse/ACM-10400--versions removed

dockerymick committed:
https://issues.redhat.com/browse/ACM-10483

jc-berger committed:
Merge pull request #6286 from stolostron/jcberger-10400-remove-hidden-comments

removed hidden comments and other fixes

cadawson committed:
fix link

cadawson committed:
Merge pull request #6294 from stolostron/cadawson-patch-1

fix link

cadawson committed:
fix link extra #

cadawson committed:
Merge pull request #6295 from stolostron/cadawson-patch-2

fix link extra #

cadawson committed:
link fix

cadawson committed:
Merge pull request #6296 from stolostron/cadawson-patch-2

link fix

cadawson committed:
fix link

cadawson committed:
Merge pull request #6297 from stolostron/cadawson-patch-2

fix link

cadawson committed:
fix conref

cadawson committed:
Merge pull request #6298 from stolostron/cadawson-patch-2

fix conref

xenolinux committed:
NoJira: Update Tech to Technology

xenolinux committed:
Merge pull request #6299 from stolostron/update-tech-technology

NoJira: Update Tech to Technology

oafischer committed:
https://issues.redhat.com/browse/ACM-10553 Update must-gather (#6283)

dockerymick committed:
Increasing and decreasing pv and pvc 2.10 (#6287)

* Increasing and decreasing pv and pvc 2.10

* Apply suggestions from code review

accepting suggestion

Co-authored-by: jc-berger <70717303+jc-berger@users.noreply.github.com>

* Update observability/customize_observability.adoc

* Update observability/customize_observability.adoc

* Apply suggestions from code review

---------

Co-authored-by: jc-berger <70717303+jc-berger@users.noreply.github.com>

dockerymick committed:
https://issues.redhat.com/browse/ACM-10649 (#6293)

* https://issues.redhat.com/browse/ACM-10649

* https://issues.redhat.com/browse/ACM-10649

* revert command change

dockerymick committed:
Update compliance_history.adoc (#6300)

quick change for alignment 
https://issues.redhat.com/browse/ACM-10649

jc-berger committed:
fixedbroken link (#6284)

* fixedbroken link

* turned xref into a link

* reverted back to xref, troubleshooting in pantheon

* changed anchor to name of file

* changed anchor to be consistent with other anchors in the overview list

* added backup global hub to main doc

---------

Co-authored-by: jc-berger <>

dockerymick committed:
Fixing book (#6301)

dockerymick committed:
Merge branch '2.10_prod' into 2.10_stage

cadawson reviewed (approved):
Looks good! Thanks!

swopebe committed:
Merge pull request #6288 from stolostron/swopebe-patch-3

https://issues.redhat.com/browse/ACM-10400 Update release_notes_intro.adoc wrong version in removal

swopebe committed:
Merge pull request #6289 from stolostron/swopebe-patch-4

https://issues.redhat.com/browse/ACM-10400 Update release_notes_intro…

swopebe committed:
Merge pull request #6290 from stolostron/swope-version-updates

https://issues.redhat.com/browse/ACM-10400--versions removed

dockerymick committed:
https://issues.redhat.com/browse/ACM-10023 (#6267)

* https://issues.redhat.com/browse/ACM-10023

* Update governance/install_operator.adoc

* Update whats_new.adoc

* Apply suggestions from code review

Accepting suggestions after dev review

Co-authored-by: Justin Kulikauskas <44813129+JustinKuli@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: jc-berger <70717303+jc-berger@users.noreply.github.com>

* Apply suggestions from code review

accepting suggestion from dev

Co-authored-by: Justin Kulikauskas <44813129+JustinKuli@users.noreply.github.com>

---------

Co-authored-by: Justin Kulikauskas <44813129+JustinKuli@users.noreply.github.com>
Co-authored-by: jc-berger <70717303+jc-berger@users.noreply.github.com>

swopebe committed:
https://issues.redhat.com/browse/ACM-10400--versions and other updates to older content based on developer feedback

dockerymick committed:
https://issues.redhat.com/browse/ACM-10483 (#6291)

* updates

* Updates

* Update policy_overview.adoc

Updates after dev review

* Update governance/policy_overview.adoc

Co-authored-by: Gus Parvin <gparvin@redhat.com>

* Update governance/policy_overview.adoc

* Update governance/policy_overview.adoc

---------

Co-authored-by: Gus Parvin <gparvin@redhat.com>

swopebe committed:
Merge pull request #6307 from stolostron/swope-install4

https://issues.redhat.com/browse/ACM-10400--versions and other update…

dhaiducek committed:
Add note about eventing change (#6264)

Signed-off-by: Dale Haiducek <19750917+dhaiducek@users.noreply.github.com>

swopebe committed:
Update rbac_implement_rhacm.adoc

swopebe committed:
https://issues.redhat.com/browse/ACM-10750--rhel-version-change

swopebe committed:
Merge pull request #6271 from mprahl/fix-rbac

Fix the Governance RBAC documentation

swopebe committed:
Merge pull request #6308 from stolostron/swope-mustgather

https://issues.redhat.com/browse/ACM-10750--rhel-version-change

oafischer committed:
Merge pull request #6282 from stolostron/of-10170

https://issues.redhat.com/browse/ACM-10170 Remove YAML switch
